pkg/scheduler/framework/runtime/framework.go

```diff
@@ -628,9 +626,8 @@ func (f *frameworkImpl) RunScorePlugins(ctx context.Context, state *framework.Cy
 		}
```

Comment:
above: `errCh.SendErrorWithCancel(status.AsError(), cancel)`.

However, we should also preserve the plugin name so we can add it to `ErrorS`. But we can leave this for a followup, as part of #94695.